app/helpers/application_helper/title.rb

```diff
@@ -59,10 +53,6 @@ def title_from_layout(layout)
         _("Most Recent Monitor Alerts")
       when "monitor_alerts_overview"
         _("Monitor Alerts Overview")
-      when "network_topology"
-        _("Network Topology")
-      when "physical_infra_topology"
-        _("Physical Infrastructure Topology")
```

Comment:
Wow, this method.  😭   At least, it's a little less complicated now.  It's like trying to find Waldo looking for the `when` clause that doesn't follow the convention.